ui/pages/create-account/connect-hardware/select-hardware.js

```diff
@@ -129,6 +152,7 @@ export default class SelectHardware extends Component {
         return this.renderLedgerTutorialSteps();
       case 'trezor':
         return this.renderTrezorTutorialSteps();
+      case 'lattice': // TODO: Add Lattice tutorial
```

Comment:
Do you intent on resolving this TODO before this PR is merged? Or is this a reminder for the future?

Reply:
This is up to your design team. We didn't want to use the existing Ledger/Trezor asset because it showed a USB device being inserted into a laptop.

We're more than happy to leave the tutorial section empty and have this note as a future reminder but we can also coordinate on an asset if that's important on your side.

Reply:
Where in the design is the Lattice tutorial? I don't see it in the screenshots

Reply:
There is no Lattice tutorial at the moment. This line of code doesn't do anything and was just meant to be a placeholder for discussion.

From the GridPlus perspective we don't **need** a tutorial, but if you all want us to have one we're happy to work with you on that. I do think we would need a new asset to match the style of the Ledger/Trezor one -- we don't have anything internally that would work.